connection.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
```

Comment:
Create a server on NodeJs and create two collections in mongodb ****(don’t use mongoose)**** Users and UsersProfile